dvc/info.py

```diff
@@ -20,14 +21,19 @@
 else:
     package = f"({PKG})"
 
+if sys.version_info < (3, 8):
+    import importlib_metadata
+else:
+    import importlib.metadata as importlib_metadata
+
 
 def get_dvc_info():
     info = [
         f"DVC version: {__version__} {package}",
         "---------------------------------",
         f"Platform: Python {platform.python_version()} on "
         f"{platform.platform()}",
-        f"Supports: {_get_supported_remotes()}",
+        f"Plugins:{_get_supported_remotes()}",
```

Comment:
Let's keep `Supports` for now, since we don't have plugins just yet and if we introduce this new term we'll have to change docs too.
```suggestion
        f"Supports: {_get_supported_remotes()}",
```